### PR TITLE
Move DRBD habootstrap template and notify script of exporter to drbd-formula

### PR DESCRIPTION
**Pull request by nick-wang**
Move the DRBD habootstrap formula template and notify script
of ha exporter to drbd-formula.

Should use together with https://github.com/SUSE/drbd-formula/pull/21, drbd-formula >= 0.4.2

As long as this pull request merged. All changes in habootstrap cluster template should 
change in drbd-formula:templates/habootstrap-formula/cluster_resources_nfs_cloud.j2

### CODE REVIEW

File: pillar_examples/automatic/drbd/cluster.sls [hunk 1]
```diff
@@ -47,7 +47,7 @@ cluster:
       {% endif %}
     method: 'update'
     template:
-      source: salt://drbd_node/files/templates/drbd_cluster.j2
+      source: /usr/share/salt-formulas/states/drbd/templates/habootstrap-formula/cluster_resources_nfs_cloud.j2
```
Comment (arbulu89) [inline pillar_examples/automatic/drbd/cluster.sls:50]: Let's put `salt://drbd/templates/habootstrap-formula/cluster_resources_nfs_cloud.j2` here.
Here `salt` refers to the root_files we have in the salt configuration file, and `/usr/share/salt-formulas/states` is already there

File: salt/drbd_node/custom_handlers.sls [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # install the custom handler for splitbrain
 /usr/lib/drbd/notify-split-brain-haclusterexporter-suse-metric.sh:
   file.managed:
-    - source: salt://drbd_node/files/notify-split-brain-haclusterexporter-suse-metric.sh
+    - source: /usr/share/salt-formulas/states/drbd/templates/ha_cluster_exporter/notify-split-brain-haclusterexporter-suse-metric.sh
```
Comment (arbulu89) [inline salt/drbd_node/custom_handlers.sls:4]: The same for `salt` thing: `salt://drbd/templates/ha_cluster_exporter/notify-split-brain-haclusterexporter-suse-metric.sh`

**Review by arbulu89 — CHANGES_REQUESTED** [review]
Thanks @nick-wang Some comments below.
Besides, is the PR removing the whole `files` folder in `drbd_node` folder?

File: pillar_examples/automatic/drbd/cluster.sls (hunk 1 above)
Comment (nick-wang) [reply pillar_examples/automatic/drbd/cluster.sls]: Thanks, i totally forget this tips.
It looks better to remove the absolute path.

**Review by arbulu89 — APPROVED** [review]
Thanks @nick-wang ,
If you tested (at least in 1 provider) and works go ahead and merge

**Comment by nick-wang** [comment]
> Thanks @nick-wang ,
> If you tested (at least in 1 provider) and works go ahead and merge

Yes, i tested this with new drbd-formula package on libvirt (against the latest develop) already.
It works as expected, so i will merge them.

Thanks a lot for the review.

**Pull request merged**